stingray/utils.py

```diff
@@ -13,6 +15,11 @@ def jit(fun):
         return fun
 
 
+class UnrecognizedMethod(Exception):
+    pass
+
+
+
```

Comment:
Sure, I understand the importance and general use case. I did not understand why we need this here. Although we'd need it at some point of time in future, so no worries.